include/xeus-zmq/xserver_zmq.hpp

```diff
@@ -10,11 +10,19 @@
 #ifndef XEUS_SERVER_ZMQ_HPP
 #define XEUS_SERVER_ZMQ_HPP
 
+#ifndef UVW_AS_LIB
+#define UVW_AS_LIB
+#include <uvw.hpp>
```

Comment:
Shouldn't we include this header even if `UVW_AS_LIB` is not defined? Otherwise, a short comment explaining how uvw primitives are found could be helpful.

Reply:
I moved these `UVW_AS_LIB` directives to the CMakeLists file as `target_compile_definitions()` instead, to make it cleaner and to include uvw every time it's needed without compilation errors.